src/test/java/com/iopipe/__WrappedConnectionFactory__.java

```diff
@@ -115,8 +115,8 @@ public RemoteResult send(RequestType __t, RemoteRequest __r)
 			throws NullPointerException, RemoteException
 		{
 			// Debug
-			Logger.debug("Snooped request ({}): {}.", __t,
-				__r.bodyAsString());
+			/*Logger.debug("Snooped request ({}): {}.", __t,
```

Comment:
Why comment vs delete?

Reply:
Might be useful to un-comment this in the future if it is needed to be done, something just temporary.